cryptography/src/bls12381/primitives/ops.rs

```diff
@@ -2383,9 +2383,12 @@ mod tests {
 
             // Initialize the evaluation point and the index.
             let mut eval_at_scalar = Scalar::zero();
-            eval_at_scalar.set_int(eval_at_x + 1);
+            eval_at_scalar.set_int(eval_at_x);
```

Comment:
This is a clever fix 🙏

Reply:
Opted to add an initializer from `u64` to avoid performing ops in "scalar space".